Cargo.toml

```diff
@@ -19,17 +19,15 @@ curve25519-dalek = "3.0.0"
 bs58 = "0.4.0"
 sha2 = "0.9"
 getrandom = { version = "0.2", features = ["js"] }
-pairing-plus = "0.19"
-bbs = { version = "0.4.1", default-features = false }
 hkdf = "0.11"
 arrayref = "0.3"
 did_url = "0.1.0"
 serde = "1.0"
 serde_json = "1.0"
 base64 = "0.13"
 libsecp256k1 = "0.5.0"
-# remove after bbs version is updated with latest blake2 and generic-array >= 14
-generic-array = "0.12.4"
+bls = { git = "https://github.com/mikelodder7/bls-signatures" }
```

Comment:
have you considered publish implementations? This one looks promising: https://crates.io/crates/signature_bls
Git ref will prevent this crate from been published itself, and that's not ideal =)

Reply:
Seems to be a near identical drop in. Thanks for suggesting that.